circuit_knitting/utils/transforms.py

```diff
@@ -51,11 +51,12 @@ class SeparatedCircuits(NamedTuple):
     ``qubit_map`` is a list with length equal to the number of qubits in the original circuit.
     Each element of that list is a 2-tuple which includes the partition label
     of that qubit, together with the index of that qubit in the corresponding
-    subcircuit.
+    subcircuit.  If the original qubit is unused and has been removed from the separated circuits,
+    then that tuple will be equal to ``(None, None)``.
     """
 
     subcircuits: dict[Hashable, QuantumCircuit]
-    qubit_map: list[tuple[Hashable, int]]
+    qubit_map: list[tuple[Hashable, int] | tuple[None, None]]
```

Comment:
It turns out to be a lot easier to work with this if it's always a 2-tuple (which means you can always unpack it), which is which I've used `(None, None)` instead of `None`.